Calendar to OBR Store

ManuelLovell opened the pull request:
Make sure that your submission has the following:

- [x] A link to the raw github markdown file with a YAML front matter definition containing:
  - [x] The title of your extension
  - [x] A description of your extension
  - [x] The author
  - [x] A hero image
  - [x] An icon
  - [x] Tags (these will help with your extension's discoverability)
  - [x] A link to your extensions manifest

Please, go through these steps before you submit a PR.

- [x] You have done your changes in a separate branch.

- [x] You have a descriptive commit message with a short title (first line).

- [x] You have only one commit (if not, squash them into one commit).

- [x] Your pull request MUST target the `main` branch on this repository.

- [x] Your pull request puts your extensions details as the last entry in the extensions.json file

nthouliss commented:
Heya! Sorry about the wait, you have a space at the top of your markdown file so that will need to be removed in order to get the information correctly. 

There's also a 404 on your What's New section of the extension

ManuelLovell commented:
> Heya! Sorry about the wait, you have a space at the top of your markdown file so that will need to be removed in order to get the information correctly.

Fixed!
> 
> There's also a 404 on your What's New section of the extension
Whoopsied on some capitals. Fixed.
